components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -189,10 +189,13 @@ object typing {
 
   }
 
-  case class TypedClass private[typing] (klass: Class[_], params: List[TypingResult]) extends SingleTypingResult {
```

Comment:
Why we have now `value` in both `TypedClass` and `TypedObjectWithValue`?

Reply:
:thinking:

Reply:
yeah i've completely missed the opportunity to use `TypedObjectWithValue`, i've changed it and force-pushed a much-reduced PR